cmd/substreams-sink-postgres/inject_csv.go

```diff
@@ -0,0 +1,239 @@
+package main
+
+import (
+	"context"
+	"encoding/csv"
+	"fmt"
+	"regexp"
+	"strconv"
+	"strings"
+	"time"
+
+	"github.com/jackc/pgx/v4/pgxpool"
+	_ "github.com/lib/pq"
+	"github.com/spf13/cobra"
+	"github.com/spf13/pflag"
+	. "github.com/streamingfast/cli"
+	"github.com/streamingfast/dstore"
+	sink "github.com/streamingfast/substreams-sink"
+
+	"github.com/streamingfast/substreams-sink-postgres/db"
+	"go.uber.org/zap"
+)
+
+var injectCSVCmd = Command(injectCSVE,
+	"inject-csv <schema> <input-path> <table> <psql-dsn> <start-block> <stop-block>",
+	"Injects generated CSV rows for <table> into the database pointed by <psql-dsn> argument. Can be run in parallel for multiple rows up to the same stop-block. Watch out, the start-block must be aligned with the range size of the csv files or the module inital block",
```

Comment:
The short description of the command should be one "short" line. Define a `Description` builder to add long description with longer details.
```suggestion
	"Injects generated CSV rows for <table> into the database pointed by <psql-dsn> argument",
	Description(`
        Can be run in parallel for multiple rows up to the same stop-block. 
	    
        Watch out, the start-block must be aligned with the range size of the csv files or the module inital block",
	`),
```

Reply:
Done